_config.yml

```diff
@@ -83,6 +83,14 @@ theme: aframe
 exclude_generator:
 - index
 
+# Feed
+## Plugin https://github.com/hexojs/hexo-generator-feed
+feed:
+  type: atom
+  path: feed.xml
```

Comment:
Probably easier for existing subscribers to keep using the same URL, and just provide one good feed instead of several.